src/tss2-fapi/api/Fapi_Provision.c

```diff
@@ -1407,7 +1407,10 @@ Fapi_Provision_Finish(FAPI_CONTEXT *context)
             } else {
                 /* No certificate was stored in the TPM and ek_cert_less was not set.*/
                 goto_error(r, TSS2_FAPI_RC_NO_CERT,
-                           "No certifcate was stored in the TPM.", error_cleanup);
+                           "No EK certifcate found for current crypto profile found. "
+                           "You may want to switch the profile in fapi-config or "
+                           "set the ek_cert_less or ek_cert_file options in fapi-config.",
+                           error_cleanup);
```

Comment:
One thought, what if had a link to the github io writeup here? The links should be stable for a long time.